package.json

```diff
@@ -124,12 +124,13 @@
     "ng2-bootstrap": "1.3.3",
     "ng2-restangular": "0.1.30",
     "ngx-base": "1.2.9",
-    "ngx-dropdown": "0.0.22",
+    "ngx-bootstrap": "1.7.1",
```

Comment:
I left that in place because there are other components being used such as ModalModule, TabsModule, etc. The planner also uses components like CollapseModule, so ng2-bootstrap is required until all that is updated as well. Not certain what the runtime console is using, yet?

I'm familiar with the changes for the new dropdown layout; thus, I wanted to limit the scope of these changes to sync with pattenrfly-ng. The PR was getting large as it is...